### PR TITLE
Improvements to CERT-EU CSV parser.

### PR DESCRIPTION
**Pull request by antoinet**
Hello Pierre

I'm not really sure if I'm doing this right: I'd like to add my improvements to https://github.com/certtools/intelmq/pull/1287, I think you first have to approve my PR into your repo for this.

### CODE REVIEW

File: intelmq/bots/parsers/cert_eu/parser_csv.py [hunk 1]
```diff
@@ -26,47 +27,52 @@ class CertEUCSVParserBot(ParserBot):
...
-        # event["?"] = line["first_seen"]
-        # event["?"] = line["num_sensors"]
-        event["feed.accuracy"] = line["confidence level"]  # TBC is 0<=x<=100 ?
-        # event["?"] = line["last seen"]
-        event["event_description.target"] = line["target"]  # TBC
-        event["source.url"] = line["phishing"]  # TBC or destination.url?
-        event["source.asn"] = line["asn"]  # TBC or destination?
-        event["source.fqdn"] = line["domain name"]  # TBC or destination?
+        if "datasource" in line:
+            extra["datasource"] = line["datasource"]
```
Comment (wagner-certat) [inline intelmq/bots/parsers/cert_eu/parser_csv.py]: Just use the field `extra.datasource`

File: intelmq/bots/parsers/cert_eu/parser_csv.py [hunk 2]
```diff
@@ -26,47 +27,52 @@ class CertEUCSVParserBot(ParserBot):
...
+        event.add("source.ip", line["source ip"])
+        event.add("time.observation", line["observation time"])
+        event.add("tlp", line["tlp"])
+        event.add("event_description.text", line["description"])
+        event.add("classification.type", self.abuse_to_intelmq[line["type"]])
+        if "count" in line:
+            extra["count"] = line["count"]
```
Comment (wagner-certat) [inline intelmq/bots/parsers/cert_eu/parser_csv.py]: can be converted to int

File: intelmq/bots/parsers/cert_eu/parser_csv.py (hunk 1 above)
Comment (antoinet) [reply intelmq/bots/parsers/cert_eu/parser_csv.py]: you mean `event['extra.datasource']`?
this is applicable for all `extra` fields?

Comment (wagner-certat) [reply intelmq/bots/parsers/cert_eu/parser_csv.py]: sure

**Pull request merged**